corgi-editor/corgi-editor.el

```diff
@@ -48,6 +48,11 @@
 
 (use-package undo-fu)
 
+(use-package flymake
+  :config
+  (remove-hook 'flymake-diagnostic-functions #'flymake-proc-legacy-flymake)
+  (add-hook 'prog-mode-hook (lambda () (flymake-mode +1))))
```

Comment:
We don't want to force everyone to install this, especially if turning it on in every mode. I tried this out and started getting warnings in an emacs lisp file. I definitely do not want that.

Can we move this over to the kondo package, and only enable it in clojure/clojurescript buffers?

Reply:
I will remove the add-hook to prog-mode-hook so that flymake is not loaded when you start a mode derived from prog-mode. However, I think we should leave the use-package stanza in corgi-editing as some people might want to use flymake with other modes (i.e. with javascirpt in particular). Without the add-hook prog-mode-hook expression, flymake should only be loaded either if corgi-clojoure-kondo is loaded (for clojure/clojurescript) or if flymake-mode is added to a mode hook (like js2-mode-hook) or if the user does M-x flymake-mode.